netci.groovy

```diff
@@ -1500,57 +1613,72 @@ def static addTriggers(def job, def branch, def isPR, def architecture, def os,
                     break
             }
             break
+
         // editor brace matching: }
         case 'x86': // editor brace matching: {
-            assert ((os == 'Windows_NT') || ((os == 'Ubuntu') && (scenario == 'default')))
+            assert ((os == 'Windows_NT') || ((os == 'Ubuntu') && isNormalOrInnerloop))
             if (os == 'Ubuntu') {
                 // Triggers on the non-flow jobs aren't necessary here
                 if (!isFlowJob) {
                     break
                 }
+                
                 // on-demand only for ubuntu x86
                 Utilities.addGithubPRTriggerForBranch(job, branch, "${os} ${architecture} ${configuration} Build",
                     "(?i).*test\\W+${os}\\W+${architecture}\\W+${configuration}.*")
                 break
+
             }
             switch (scenario) {
-                case 'default':
-                    if (configuration == 'Checked') {
-                        assert !job.name.contains("centos")
+                case 'innerloop':
+                    if (configuration == 'Checked' || configuration == 'Release') {
                         Utilities.addGithubPRTriggerForBranch(job, branch, "${os} ${architecture} ${configuration} Innerloop Build and Test")
                     }
                     else {
                         Utilities.addGithubPRTriggerForBranch(job, branch, "${os} ${architecture} ${configuration} Build and Test",
                             "(?i).*test\\W+${os}\\W+${architecture}\\W+${configuration}.*")
                     }
                     break
+
+                case 'normal':
+                    if (configuration == 'Checked') {
+                        Utilities.addGithubPRTriggerForBranch(job, branch, "${os} ${architecture} ${configuration} Build and Test",
+                            "(?i).*test\\W+${os}\\W+${architecture}\\W+${configuration}.*")
```

Comment:
This trigger phrase looks like a subset of the ones below (e.g., ilrt). So if you trigger an ilrt run, you'll also get this. Seems like the phrase needs to include "Build and Test" at the end.